### PR TITLE
Adding documentation for Somfy MyLink platform

### PR DESCRIPTION
**Pull request by bendews**
**Description:**
This adds the documentation required for the Somfy MyLink platform based on the Synergy JsonRPC API.
The following component pages are included:
- compenent/somfy_mylink
  - cover/somfy_mylink
  - scene/somfy_mylink

It might be a good idea to change the name of the current `tahoma.png` logo to `somfy.png` at some point, however in the spirit of introducing as little change at a time, I'm leaving it as `tahoma.png`.

Thanks!

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17654

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/cover.somfy_mylink.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: "Somfy MyLink Cover"
+description: "Instructions on how to integrate Somfy MyLink covers into Home Assistant."
+date: 2018-10-20 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Cover
+ha_release: 0.XX
```
Comment (frenck) [inline source/_components/cover.somfy_mylink.markdown]: Please use the next major version here (no placeholders).

File: source/_components/somfy_mylink.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+title: "Somfy MyLink"
+description: "Instructions on how to integrate Somfy MyLink devices with Home Assistant."
+date: 2018-10-20 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Hub
+ha_release: 0.XX
```
Comment (frenck) [inline source/_components/somfy_mylink.markdown]: Please use the next major version here (no placeholders).

File: source/_components/somfy_mylink.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+The `Somfy MyLink` component platform is used as an interface to a compatible Somfy MyLink hub utilizing the `Synergy` API. It adds covers and scenes from the Somfy MyLink platform.
+
+To use your compatible `Somfy MyLink` devices in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+somfy_mylink:
+  host: 10.1.1.100
+  password: mylink_id
+  cover_options:
```
Comment (frenck) [inline source/_components/somfy_mylink.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/somfy_mylink.markdown [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Hub
+ha_release: 0.XX
+ha_iot_class: "Assumed State"
+---
+
```
Comment (frenck) [inline source/_components/somfy_mylink.markdown:15]: One empty line to many. Please remove one.

**Review by frenck — CHANGES_REQUESTED** [review]
@bendews I've reviewed your PR and left some comments. Could you please take a look?
Furthermore, I don't get the Tahoma logo. This is brand new docs about Somfy, why not add the Somfy logo?

File: source/_components/somfy_mylink.markdown (hunk 4 above)
Comment (bendews) [reply source/_components/somfy_mylink.markdown:15]: Have removed in new commit

File: source/_components/somfy_mylink.markdown (hunk 3 above)
Comment (bendews) [reply source/_components/somfy_mylink.markdown]: Have removed and put in another more advanced example, based off of the `nest` documentation: https://www.home-assistant.io/components/nest/

File: source/_components/cover.somfy_mylink.markdown (hunk 1 above)
Comment (bendews) [reply source/_components/cover.somfy_mylink.markdown]: Have updated all files with v0.81

**Comment by bendews** [comment]
@frenck Massive thanks for the review! I have hopefully remediated the items you have raised.
Regarding the Tahoma logo, Tahoma is a Somfy brand, so `tahoma.png` is actually just the Somfy logo, you can see this here: https://github.com/home-assistant/home-assistant.io/blob/next/source/images/supported_brands/tahoma.png

File: source/_components/somfy_mylink.markdown [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+To use your compatible `Somfy MyLink` devices in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+somfy_mylink:
+  host: 10.1.1.100
+  password: mylink_id
+```
+
+```yaml
```
Comment (frenck) [inline source/_components/somfy_mylink.markdown]: Please move the second example, after the configuration list.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/somfy_mylink.markdown (hunk 5 above)
Comment (bendews) [reply source/_components/somfy_mylink.markdown]: @frenck, I've moved this. Sorry for the delay, the GitHub outage has been playing havoc with my commits.

Comment (bendews) [reply source/_components/somfy_mylink.markdown]: @frenck, I've moved this. Sorry for the delay, the GitHub outage has been playing havoc with my commits.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for updating the PR @bendews! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by bendews** [comment]
@frenck Thanks for the approval! Any chance we could nudge someone to review the component? 😁

**Comment by frenck** [comment]
@bendews No, should not do that. It is busier this time of year, just be patient.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**